packages/application-audit-fallback/src/handlers/handleMessage.ts

```diff
@@ -0,0 +1,29 @@
+import { logger, Message } from "pagopa-interop-commons";
+import { config } from "../config/config.js";
+import { decodeSQSMessage } from "../models/queue.js";
+import { KafkaProducer } from "../models/kafka.js";
+
+export function handleMessage(producer: KafkaProducer) {
+  return async function processMessage(message: Message): Promise<void> {
+    const decodedMessage = decodeSQSMessage(message);
+
+    const loggerInstance = logger({
+      serviceName: config.serviceName,
+      spanId: decodedMessage.spanId,
+      correlationId: decodedMessage.correlationId,
+    });
+
+    await producer.send({
+      messages: [
+        {
+          key: decodedMessage.correlationId,
+          value: JSON.stringify(decodedMessage),
+        },
+      ],
+    });
+
+    loggerInstance.info(
+      "Application audit sent to Kafka topic through fallback path"
```

Comment:
let's add the `serviceName`, `correlationId` and `spanId` in the log to have more info